type-definitions/immutable.js.flow

```diff
@@ -32,6 +32,21 @@
 // some constructors and functions.
 type PlainObjInput<K, V> = {[key: K]: V, __proto__: null};
 
+// Helper types to extract the "keys" and "values" use by the *In() methods.
+// Exported for experimental use only, may change.
+export type $KeyOf<C> = $Call<
+  & (<K>(?_Collection<K, *>) => K)
+  & (<T>(?RecordInstance<T>) => $Keys<T>),
+  C
+>;
+
+export type $ValOf<C, K = *> = $Call<
```

Comment:
Would `K = $KeyOf<C>` work here? `K = string` may also work well.

Reply:
Great idea to use `$KeyOf<C>` as a default